packages/one-app-bundler/webpack/loaders/common.js

```diff
@@ -67,6 +67,7 @@ const babelLoader = (babelEnv) => ({
   options: {
     extends: path.join(packageRoot, '.babelrc'),
     envName: babelEnv,
+    cacheDirectory: path.join(packageRoot, '.babel-cache'),
```

Comment:
Should we rename this to be generic so in the future we can add more stuff in there? Or do we want a cache folder specific for each thing?

Reply:
changed this to `.build-cache`